src/utils/wallet-seed.js

```diff
@@ -111,3 +111,13 @@ async function deriveEncryptionKey(password, salt, iterations, digest) {
 export function lockWallet() {
   setUnlockedMnemonicAndSeed(null, null);
 }
+
+export function forgetWallet() {
```

Comment:
Need to add 

```
  localStorage.removeItem('walletPrivateKeyImports');
```
And 
```
unlockedMnemonicAndSeed = { mnemonic: null, seed: null, importsEncryptionKey: null };
```

Reply:
Dumb question: is there a reason not to use https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear ?

Reply:
Added the above changes.